src/components/tutorial-card/helpers/format-tutorial-card.ts

```diff
@@ -4,7 +4,7 @@ import {
   ProductUsed,
   TutorialLite as ClientTutorialLite,
 } from 'lib/learn-client/types'
-import { TutorialCardPropsWithId } from './types'
+import { TutorialCardPropsWithId } from '../types'
```

Comment:
Moved this to a `helpers` folder for now; will move to `lib/tutorials` in a follow-up PR. See 🎟️ [[chore] make lib/tutorials for shared use tutorial helpers](https://app.asana.com/0/1202097197789424/1202180048371484/f)